qplist.pro

```diff
@@ -0,0 +1,14 @@
+QT += xml
```

Comment:
I have thought about that very long, and first I totally agreed with you.

On the other hand, I came to the conclusion that pods simply aren't Qt modules, because they do not belong to Qt. If you were honestly strict about naming, it should be called "plist". Because only Qt modules start with Qt and Qt classes start with a Q, custom classes have either your own prefix (in KDE all classes started with a "K" for example, like "KProcess" instread of "QProcess") or any at all.

It's really a difficult question to say what's the right thing and I ended up just prefixing with "q". You can leave your repository name as it is, but when it is cloned as a submodule it will be called "qplist".